### ISSUE TITLE
template wtrans workshop

### ISSUE THREAD

**Issue by nitanilla** [nitanilla/wtrans-site#2]
http://www.bootstrapzero.com/bootstrap-templates

**Comment by nitanilla** [comment]
eu escolhi esse aqui http://bootswatch.com/simplex/

**Comment by jpivatelli** [comment]
Eu gostei desse aqui: https://w3layouts.com/preview/?l=/seen-a-corporate-business-flat-bootstrap-responsive-web-template/
Ignorem a home pois a questão da imagem não tem nada a ver. Mas a forma clean com que apresentaram pareceu interessante.

**Comment by jpivatelli** [comment]
Esse também achei legal: https://w3layouts.com/preview/?l=/bizz-wow-a-corporate-business-flat-bootstrap-responsive-web-template/
Posso mudar os ícones amarelos dos links para azuis - assim ficam combinando com a logo do wtrans :)

**Comment by nitanilla** [comment]
o primeiro q @jpivatelli  colocou pode ser... mas ainda gosto do meu kkkkk .
falta o Edgard!

**Comment by nitanilla** [comment]
@jpivatelli Jo! baixe todos esses arquivos de aqui https://github.com/nitanilla/wtrans-site/tree/master/simplex e abra on index.html no seu computador ... tente mexer nos outros controles que deixe embaixo. 

Eu fiz mais facil usando aqui: http://bootswatch.com/simplex/ 
em cada control tem uma marquinha <> com isso pode copiar o codigo para o nosso arquivo index.html.

tomei 1.20 em fazer o que ta no simplex. Me avisa se precisar ajuda :)

**Comment by jpivatelli** [comment]
Oi Pessoal,

Vou escrever por aqui que está mais fácil...
Do teste que vc fez, consegui visualizar a img em anexo. Entendi que vc
colocou a logo, foi isso? Não entendi como ficariam os links. De repente,
seria interessante fazer exatamente como está hoje para ver se funciona
legal, o que acha?

Bjs,

Em 1 de setembro de 2015 21:28, nitanilla notifications@github.com
escreveu:

> @jpivatelli https://github.com/jpivatelli Jo! baixe todos esses
> arquivos de aqui
> https://github.com/nitanilla/wtrans-site/tree/master/simplex e abra on
> index.html no seu computador ... tente mexer nos outros controles que deixe
> embaixo.
> 
> Eu fiz mais facil aqui usando aqui: http://bootswatch.com/simplex/
> em cada control tem uma marquinha <> com isso pode copiar o codigo para o
> nosso arquivo index.html.
> 
> tomei 1.20 em fazer o que ta no simplex. Me avisa se precisar ajuda :)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nitanilla/wtrans-site/issues/2#issuecomment-136902067
> .

## 

Joanna Pivatelli

**Comment by nitanilla** [comment]
não ta a imagem q voce fala @jpivatelli :(

**Comment by ghost** [comment]
Template exemplo WTRANS
- CLEI: http://clei.org/

Template Exemplo WTRANSartigos
-Template Exemplo do RePa
http://www.utdallas.edu/~supakkul/repa15/dates.html
-Template Exemplo do ICEIS
http://www.iceis.org/
Template Exemplo do CLEI

**Comment by jpivatelli** [comment]
Qual está?

2015-09-02 10:36 GMT-03:00 nitanilla notifications@github.com:

> não ta a imagem q voce fala @jpivatelli https://github.com/jpivatelli
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nitanilla/wtrans-site/issues/2#issuecomment-137082994
> .

## 

Joanna Pivatelli

**Comment by nitanilla** [comment]
@jpivatelli baixe esse aqui https://desktop.github.com/. Com isso poderemos mexer juntos no mesmo codigo do template.

**Comment by nitanilla** [comment]
@edgarsc22 @jpivatelli  O que estou tentando fazer é esse aqui http://wtrans.inf.puc-rio.br/ a tela principal do wtrans (é essa né?)

fiz melhoras, e cheguei ate aqui de acordo as suas sugestões do site do Clei
![image](https://cloud.githubusercontent.com/assets/1106435/9657348/91b50f14-5216-11e5-8368-bbed9d2454b8.png)

@jpivatelli voce poderia me dizer que items no menu poderiam ir?. ou mais ideias. sei que pelo trablaho voce nao tem acesso, mas suas ideias em texto são livres! kk
bjs

**Comment by jpivatelli** [comment]
Oi pessoal, vão algumas considerações a respeito da imagem:
1. A logo está muito grande e acaba ocupando boa parte da tela. Posso
   diminuir a logo e pensar em alguma coisa para deixá-la mais "moderna" sem
   descaracterizá-la.
2. Gostei da ideia de ser clean, mas me parece que falta algo que traga
   um pouco de cor, como umas ícones ao lado do menu ou efeito no link
   selecionado. Esse template não tinha algo nesse sentido?
3. Sugestão para a página principal do wtrans: melhorar o texto da
   http://wtrans.inf.puc-rio.br/ com a ideia de quebrá-lo e ficar mais
   atrativo para a leitura e algumas infos interessantes (como uma imagem, por
   exemplo), pensar em colocar um histórico das edições já realizadas, os
   links para as edições do wtrans e para os artigos.
4. Sugestão para os menus para as edições do wtrans: Notícias (ou
   Principal) - Organização - Fotos - ...

Em 3 de setembro de 2015 08:37, nitanilla notifications@github.com
escreveu:

> @edgarsc22 https://github.com/edgarsc22 @jpivatelli
> https://github.com/jpivatelli O que estou tentando fazer é esse aqui
> http://wtrans.inf.puc-rio.br/ a tela principal do wtrans (é essa né?)
> 
> fiz melhoras, e cheguei ate aqui de acordo as suas sugestões do site do
> Clei
> [image: image]
> https://cloud.githubusercontent.com/assets/1106435/9657348/91b50f14-5216-11e5-8368-bbed9d2454b8.png
> 
> @jpivatelli https://github.com/jpivatelli voce poderia me dizer que
> items no menu poderiam ir?. ou mais ideias. sei que pelo trablaho voce nao
> tem acesso, mas suas ideias em texto são livres! kk
> bjs
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nitanilla/wtrans-site/issues/2#issuecomment-137413514
> .

## 

Joanna Pivatelli

**Comment by jpivatelli** [comment]
Oi pessoal, vcs concordam, discordam, parcialmente...?

Em 3 de setembro de 2015 09:38, Joanna Pivatelli jpivatelli@gmail.com
escreveu:

> Oi pessoal, vão algumas considerações a respeito da imagem:
> 1. A logo está muito grande e acaba ocupando boa parte da tela. Posso
>    diminuir a logo e pensar em alguma coisa para deixá-la mais "moderna" sem
>    descaracterizá-la.
> 2. Gostei da ideia de ser clean, mas me parece que falta algo que
>    traga um pouco de cor, como umas ícones ao lado do menu ou efeito no link
>    selecionado. Esse template não tinha algo nesse sentido?
> 3. Sugestão para a página principal do wtrans: melhorar o texto da
>    http://wtrans.inf.puc-rio.br/ com a ideia de quebrá-lo e ficar mais
>    atrativo para a leitura e algumas infos interessantes (como uma imagem, por
>    exemplo), pensar em colocar um histórico das edições já realizadas, os
>    links para as edições do wtrans e para os artigos.
> 4. Sugestão para os menus para as edições do wtrans: Notícias (ou
>    Principal) - Organização - Fotos - ...
> 
> Em 3 de setembro de 2015 08:37, nitanilla notifications@github.com
> escreveu:
> 
> > @edgarsc22 https://github.com/edgarsc22 @jpivatelli
> > https://github.com/jpivatelli O que estou tentando fazer é esse aqui
> > http://wtrans.inf.puc-rio.br/ a tela principal do wtrans (é essa né?)
> > 
> > fiz melhoras, e cheguei ate aqui de acordo as suas sugestões do site do
> > Clei
> > [image: image]
> > https://cloud.githubusercontent.com/assets/1106435/9657348/91b50f14-5216-11e5-8368-bbed9d2454b8.png
> > 
> > @jpivatelli https://github.com/jpivatelli voce poderia me dizer que
> > items no menu poderiam ir?. ou mais ideias. sei que pelo trablaho voce nao
> > tem acesso, mas suas ideias em texto são livres! kk
> > bjs
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/nitanilla/wtrans-site/issues/2#issuecomment-137413514
> > .
> 
> ## 
> 
> Joanna Pivatelli

## 

Joanna Pivatelli

**Comment by nitanilla** [comment]
@jpivatelli @edgarsc22 
mudanças feitas..
OBS: o Logo se vê grande pois não tem muito na pagina ainda, então o template tenta fazer um resize..

![image](https://cloud.githubusercontent.com/assets/1106435/9683090/1efe6bd2-52e3-11e5-9ac2-f1128586ef0e.png)

esse aqui fica pendente para o site de cada edição de wtrans
1. Sugestão para os menus para as edições do wtrans: Notícias (ou
   Principal) - Organização - Fotos - ...

mais sugestões?
